app/Services/GitHub/Connection/GitHubConnectionManager.php

```
@@ -0,0 +1,219 @@
+    public function __construct(
+        private CacheRepository $cache,
+        private ConfigRepository $config,
+    ) {}
+
+    public function status(?string $connection = null): GitHubConnectionStatus
+    {
+        $connection = $this->normalizeConnection($connection);
+        $config     = $this->config->get("github.connections.{$connection}");
+
+        if (! is_array($config)) {
+            return GitHubConnectionStatus::misconfigured("GitHub connection [{$connection}] is not configured.");
```

PHPMD (2.15.0)

Avoid using static access to class 'App\Services\GitHub\Connection\GitHubConnectionStatus' in method 'status'. (undefined)

(StaticAccess)